arch/arm/dts/tegra186.dtsi

```diff
@@ -204,12 +206,116 @@
 		#reset-cells = <1>;
 	};
 
-	bpmp-i2c {
+	gen1_i2c: i2c@3160000 {
+		compatible = "nvidia,tegra186-i2c", "nvidia,tegra114-i2c";
```

Comment:
Are you sure that the T186 I2C controller is 100% backwards-compatible with the T114 I2C controller? If so, this is fine. If not, let's only include the T186 compatible value. (A quick way to check might be to look at our downstream kernel DT files, although it's not guaranteed the requisite research/thought was put into those. Also, diff the register definition headers and see if anything changed).